pkg/api/redistribution.go

```diff
@@ -0,0 +1,39 @@
+// Copyright 2021 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package api
+
+import (
+	"github.com/ethersphere/bee/pkg/jsonhttp"
+	"github.com/ethersphere/bee/pkg/tracing"
+	"net/http"
+)
+
+type NodeStatusResponse struct {
+	State  string `json:"state"`
+	Round  uint64 `json:"round"`
+	Block  uint64 `json:"block"`
+	Reward string `json:"reward"`
```

Comment:
The `NodeStatusResponse` shouldn't be exported.  Also, why it is needed when it basically is the same as `storageincentives.NodeStatus` (since both have JSON tags)?

Reply:
API response required `reward` and `fee` to be in `string` that is why I have separated it.